package.json

```diff
@@ -35,7 +35,7 @@
     "test:ci": "jest --detectOpenHandles --coverage --notify --expand --silent --watchAll=false --maxWorkers=2",
     "test:silent": "npm run test -- --silent",
     "test:benchmark": "node benchmark/index.js",
-    "test:integration": "jest -c jest.config.integration.js --detectOpenHandles",
+    "test:integration": "jest -c jest.config.integration.js --detectOpenHandles --silent",
```

Comment:
We have `test:integration:silent` now (right below this script).

Reply:
Fixed